.github/workflows/ccip-ocr3-build-lint-test.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        go-version: ['1.22']
+        go-version: ['1.23.3']
```

Was `1.22` actually a stale reference to `1.22.0`? Or does that mean latest patch `1.22.x` in this context? In which case we'd want:
```suggestion
        go-version: ['1.23']
```